src/code42cli/cmds/alerts.py

```diff
@@ -186,14 +192,96 @@ def search(
         include_all, SEARCH_DEFAULT_HEADER, format
     )
     format_func = get_output_format_func(format)
+    print_events_decorator = print_events(format_func, output_header)
+    handlers = _extract_events(
```

Comment:
This is just a refactor, no big change in search. Created `_extract_events` to make sure `search` and `send-to` use the same code.